src/main/java/org/reactome/release/qa/common/AbstractQACheck.java

```diff
@@ -169,4 +171,15 @@ protected boolean isEscaped(GKInstance instance) throws Exception {
         return words;
     }
 
+    /**
+     * Checks contents of skiplist, which should be a list of skippable DbIds, to see if incoming instance should be skipped.
+     * @param inst - GKInstance, instance being checked.
+     * @param skiplistFilename - String, filename of skiplist.
+     * @return - boolean, true if inst DbId is in skiplist, false if not.
+     * @throws IOException, thrown if 'skiplistFilename' doesn't exist as a file.
+     */
+    protected static boolean inSkipList(GKInstance inst, String skiplistFilename) throws IOException {
+        List<String> skiplist = Files.readAllLines(Paths.get(skiplistFilename));
```

Comment:
Same as here: https://github.com/reactome/release-qa/pull/27/files#r520217023.

It might be worth having a method in a utility class that reads, validates, and returns db ids from a file path.  Something like `readDBIdsFromFile`?